include/oneapi/dpl/complex

```diff
@@ -24,6 +24,155 @@ namespace oneapi
 namespace dpl
 {
 using ::std::complex;
+
+using ::std::arg;
+using ::std::conj;
+using ::std::imag;
+using ::std::norm;
+using ::std::polar;
+using ::std::proj;
+using ::std::real;
+
+template <class T>
+T
+abs(const complex<T>& z)
```

Comment:
I am not sure that we want to have  20 re-call wrappers - specializations for complex type... 
In my opinion we have to make an injection into `dpl` for each function  how we made before.
Regarding the C-overloads (for float and double) with the same names - I believe that `<cmath>` is a part of the tested API as a base mathematics. It is very strange if it is not truth.

Reply:
@akukanov, what do you think about this point?

Reply:
fixed